corehq/apps/reports/dbaccessors.py

```diff
@@ -23,6 +23,13 @@ def _get_exports(domain, include_docs=True, reduce=False, **kwargs):
     )
 
 
+def get_exports_json(domain):
```

Comment:
literally the same as the function below except without the stale argument. will remove when i'm done migrating